### PR TITLE
lockfile optimized

### PR DESCRIPTION
**Pull request by bestander**
just updated lockfile that selfupdates after install

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2726,28 +2726,28 @@ istanbul-lib-hook@^1.0.7:
...
     babylon "^6.13.0"
-    istanbul-lib-coverage "^1.1.1"
+    istanbul-lib-coverage "^1.1.0"
     semver "^5.3.0"
 
-istanbul-lib-instrument@^1.7.1:
-  version "1.7.1"
-  resolved "https://registry.yarnpkg.com/istanbul-lib-instrument/-/istanbul-lib-instrument-1.7.1.tgz#169e31bc62c778851a99439dd99c3cc12184d360"
+istanbul-lib-instrument@^1.7.2:
+  version "1.7.2"
```
Comment (SimenB) [inline yarn.lock]: Why isn't this deduped away?

Comment (bestander) [inline yarn.lock]: Currently Yarn does deduping on the go "for free", this PR should address it https://github.com/yarnpkg/yarn/pull/3729.

I think deduping should be either during upgrade operations or when deduping was requested with a command.
Not sure if --force should dedupe

Comment (SimenB) [inline yarn.lock]: Will `yarn dedupe` become a thing? Right now I'm stuck manually deduping the lockfile, which really sucks.

>error The dedupe command isn't necessary. `yarn install` will already dedupe.

This is a lie

Comment (bestander) [inline yarn.lock]: Sorry, @SimenB, we just had to mitigate the issue of Yarn being too eager to dedupe the packages during any installation which resulted in unexpdected behavior.

Please chime in into this issue https://github.com/yarnpkg/yarn/issues/3778 and help out to resolve it

Comment (SimenB) [inline yarn.lock]: Ah, didn't know about that issue, thanks!

**Pull request merged**